spk/python312/crossenv/requirements-atom.txt

```diff
@@ -1,5 +1,5 @@
-pip==24.3.1
-setuptools==75.8.0
+pip==25.0.1
```

Comment:
I should have thought of this but pip 25.x has a lot of changes in it.  Maybe worth waiting a bit for this one in particular.  Applicable in all crossenv definitions.